func/Pipe/SplitPipe.sql

```diff
@@ -0,0 +1,86 @@
+-------------------------------------------------------------------
+-- SPLIT PIPE BASED ON FRACTION
+-- INPUT:	- v2_model
+-- 		    - pipe_id, fraction and interpolation (true/false)
+-- OUTPUT:	- duplicates pipe and interpolates invert levels
+-- 		    - insert new connection_node
+-------------------------------------------------------------------
+DROP FUNCTION IF EXISTS SplitPipe(integer,double precision,boolean);
+CREATE OR REPLACE FUNCTION SplitPipe (
+	split_id integer,
+    fraction double precision,
```

Comment:
om dit in lijn te krijgen met SplitChannel zou het goed zijn als tweede argument een geometry is, die Point of MultiPoint mag zijn